components/function-controller/pkg/utils/controller_utils_test.go

```diff
@@ -8,69 +24,47 @@ import (
 	"github.com/kyma-project/kyma/components/function-controller/pkg/utils"
 
 	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 func TestNewRuntimeInfo(t *testing.T) {
-
 	g := gomega.NewGomegaWithT(t)
 	cm := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "dummy",
+		},
 		Data: map[string]string{
-			"serviceAccountName": "test",
 			"dockerRegistry":     "foo",
+			"serviceAccountName": "test",
+			"runtimes":           "[]",
+			"defaults":           "{}",
+			"funcTypes":          "[]",
+			"funcSizes":          "[]",
 		},
 	}
 	ri, err := utils.New(cm)
 	g.Expect(err).NotTo(gomega.HaveOccurred())
 	g.Expect(ri.ServiceAccount).To(gomega.Equal("test"))
 	g.Expect(ri.RegistryInfo).To(gomega.Equal("foo"))
 
-	cmBroken := &corev1.ConfigMap{
+	cmMissing := &corev1.ConfigMap{
 		Data: map[string]string{
 			"serviceAccountName": "test",
 		},
 	}
-	_, err = utils.New(cmBroken)
-	g.Expect(err.Error()).To(gomega.Equal("Error while fetching docker registry info from configmap"))
-
-	cmBroken = &corev1.ConfigMap{
-		Data: map[string]string{
-			"dockerRegistry": "foo",
-		},
-	}
-	_, err = utils.New(cmBroken)
-	g.Expect(err.Error()).To(gomega.Equal("Error while fetching Service Account Name"))
+	_, err = utils.New(cmMissing)
+	g.Expect(err.Error()).To(gomega.ContainSubstring("missing config values"))
 
-	cmBroken = &corev1.ConfigMap{
+	cmBroken := &corev1.ConfigMap{
 		Data: map[string]string{
 			"serviceAccountName": "test",
 			"dockerRegistry":     "foo",
-			"runtimes":           "foo",
+			"runtimes":           "invalid",
+			"defaults":           "{}",
+			"funcTypes":          "[]",
+			"funcSizes":          "[]",
 		},
 	}
 	_, err = utils.New(cmBroken)
-	g.Expect(err.Error()).To(gomega.ContainSubstring("unmarshal"))
-
-}
-
-func TestDockerFileConfigMapName(t *testing.T) {
-	g := gomega.NewGomegaWithT(t)
-	runtime := "nodejs8"
-	cm := &corev1.ConfigMap{
-		Data: map[string]string{
-			"serviceAccountName": "test",
-			"dockerRegistry":     "foo",
-			"runtimes": `[
-				{
-					"ID": "nodejs8",
-					"DockerFileName": "dockerfile-nodejs8",
-				}
-			]`,
-		},
-	}
-	ri, err := utils.New(cm)
-	g.Expect(err).NotTo(gomega.HaveOccurred())
-	dockerFileCMName := ri.DockerFileConfigMapName(runtime)
```

Comment:
Where are we testing all of these?

Reply:
In the same func, this block was just relocated.